compiler/test/compilable/pragmainline2.d

```diff
@@ -2,9 +2,9 @@
 REQUIRED_ARGS: -inline -wi
 TEST_OUTPUT:
 ---
-compilable/pragmainline2.d(14): Warning: cannot inline function `pragmainline2.foo`
-compilable/pragmainline2.d(22): Warning: cannot inline function `pragmainline2.f1t`
-compilable/pragmainline2.d(25): Warning: cannot inline function `pragmainline2.f2t`
+compilable/pragmainline2.d(14): Warning: cannot inline function `pragmainline2.foo` because too costly
+compilable/pragmainline2.d(22): Warning: cannot inline function `pragmainline2.f1t` because too costly
+compilable/pragmainline2.d(25): Warning: cannot inline function `pragmainline2.f2t` because too costly
```

Comment:
There's no test coverage for the new messages, existing tests only report "because too costly" which isn't accurate.